MIP22/mip22.md

```diff
@@ -140,41 +262,45 @@ The `Tinlake Flipper` takes all of the collateral from a Vault and requests rede
     }
 ```
 
-The redeem order will now be taken into account whenever loan repayments flow into the pool or other investors want to join the pool. As the pool redeems the DROP any user can call the function `take`. It moves the DAI available for withdrawal from the pool and either repays up outstanding debt (`tab`) to the `vow` or returns it to the pool. This method can be called multiple time to cause the pool to `disburse` more DAI and return it to the pool as funds available for borrowers.
+As DAI is redeemed from the Pool, the manager transfers the DAI into the `vow` until the `tab` is recovered. At that point any additional DAI that could be redeemed with the DROP is given back to the Pool.
```

Comment:
I'll toss this comment here, but I noticed we need to make that `flip.kick()` call above `LIQ-1.2` aware.  The `cat` has a `box` that fills with `litter`.  We typically empty the `litterbox` on `deal()` or `yank()` now (the latter isn't really needed) by calling `cat.claw(tab)`.  It's a quick fix.

Here is a good example of how to make the update:
https://github.com/livnev/rwa-example/pull/1

This is a quick fix to apply to `MIP22`, but should also be tested for in the deploy integration tests.